### PR TITLE
feat: Alert feature

### PR DESCRIPTION
**Pull request by jDineshKarthic**
Alert feature added

### CODE REVIEW

File: components/Alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    gap: 1rem;
+    padding: 1rem;
+    max-width: fit-content;
+    border-radius: 0.6rem;
+    box-shadow: var(--primary-box-shadow);
+}
+
+.al-error {
+    background-color: var(--lt-pink);
+    /* var(--lt-pin); */
```
Comment (srejitk) [inline components/Alerts/alert.css:13]: Remove the inline comments if not needed

File: components/Alerts/alert.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    <title>Alerts</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Document</title>
+  </head>
+  <body>
```
Comment (srejitk) [inline components/Alerts/alert.html:20]: I don't see any code snippets or documentation. Do have a look into it. Looks good otherwise.

**Review by srejitk — APPROVED** [review]
Looks good. Have suggested few changes, have a look into it.

File: components/Alerts/alert.css (hunk 1 above)
Comment (jDineshKarthic) [reply components/Alerts/alert.css:13]: Yeah definitely

**Comment by Alimohd21** [comment]
Add a readme file and everything else looks great

**Comment by jDineshKarthic** [comment]
yeah definitely  i'll add

**Pull request merged**